src/main/java/net/mcreator/ui/action/impl/workspace/WorkspaceSettingsAction.java

```diff
@@ -174,14 +172,13 @@ public static void refactorWorkspace(MCreator mcreator, WorkspaceSettingsChange
 								.getGeneratorStats().getBaseCoverageInfo().get("model_java")
 								!= GeneratorStats.CoverageStatus.NONE) {
 							List<Model> javaModelsOld = Model.getJavaModels(mcreator.getWorkspace());
-							List<Model> javaModelsNew = Model.getJavaModels(mcreator.getWorkspace());

```

Comment:
At what point should the first one get assigned?

Reply:
Seems like a bug, but not really something to be handled in this PR

Reply:
So I can leave it as is? (With the `// TODO` comment)

Reply:
I would not even change this file as it is unrelated to VCS and this PR. List diff can come handy in other cases so this should remain in core

Reply:
Consider it done